Sources/TensorFlow/Layers/Normalization.swift

```diff
@@ -224,3 +242,165 @@ public struct LayerNorm<Scalar: TensorFlowFloatingPoint>: Layer {
     return (input - moments.mean) * inv + offset
   }
 }
+
+/// A layer that applies group normalization over a mini-batch of inputs.
+///
+/// Reference: [Group Normalization](https://arxiv.org/abs/1803.08494).
+@frozen
+public struct GroupNorm<Scalar: TensorFlowFloatingPoint>: Layer {
+  /// The offset value, also known as beta.
+  public var offset: Tensor<Scalar>
+  /// The scale value, also known as gamma.
+  public var scale: Tensor<Scalar>
+  /// The number of groups.
+  @noDerivative public let groups: Int
+  /// The axis.
+  @noDerivative public let axis: Int
+  /// The variance epsilon value.
+  @noDerivative public let epsilon: Scalar
+
+  /// Creates a group normalization layer.
+  public init(
+    offset: Tensor<Scalar>,
+    scale: Tensor<Scalar>,
+    groups: Int,
+    axis: Int,
+    epsilon: Scalar
```

Comment:
I think It's a good idea to support a default value for epsilon, as mentioned in the paper and most implementations as 1e-5.

Reply:
There's two initializers and one has default values.
I replicated `BatchNorm`'s design.
Default `epsilon` value `1e-3` is from [tfa.layers.GroupNormalization](https://github.com/tensorflow/addons/blob/v0.7.1/tensorflow_addons/layers/normalizations.py#L73-L99).